app/src/browser/config-persistence-manager.es6

```diff
@@ -29,6 +29,12 @@ export default class ConfigPersistenceManager {
       fs.copySync(templateConfigDirPath, this.configDirPath);
     }
 
+    let stat = fs.statSync(this.configDirPath);
+    let configDirPathAccessMode = (stat.mode & parseInt('777', 8)).toString(8); 
```

Comment:
The technique to extract access mode was extracted from:

https://github.com/nodejs/node-v0.x-archive/issues/3045#issuecomment-4865547